pysr/ssr.py

```diff
@@ -0,0 +1,171 @@
+import warnings
+from typing import List, Optional, Union
+
+import numpy as np
+
+from .sr import PySRRegressor
+from .utils import ArrayLike
+
+
+class PySRSequenceRegressor(PySRRegressor):
+    def __init__(
+        self,
+        recursive_history_length: int = 0,
+        **kwargs,
+    ):
+        self.recursive_history_length = recursive_history_length
+        super().__init__(**kwargs)
```

Comment:
```suggestion
        super().__init__(**kwargs)
        self.recursive_history_length = recursive_history_length
```

unless there's some reason to have it reversed? But it's standard practice to call the `super().__init__` at the top.